app/src/main/res/layout/item_status.xml

```diff
@@ -319,6 +319,16 @@
         app:layout_constraintTop_toBottomOf="@id/status_poll_description"
         app:srcCompat="@drawable/ic_reply_24dp" />
 
+    <TextView
+        android:id="@+id/status_replies"
+        android:layout_width="40dp"
+        android:layout_height="30dp"
+        app:layout_constraintBottom_toBottomOf="parent"
+        app:layout_constraintStart_toStartOf="@id/status_reply"
+        app:layout_constraintTop_toBottomOf="@id/status_poll_description"
+        android:textAlignment="viewEnd"
+        android:textSize="?attr/status_text_medium" />
```

Comment:
I think it looks best when the number is vertically centered next to the button, like this 
```suggestion
    <TextView
        android:id="@+id/status_replies"
        android:layout_width="wrap_content"
        android:layout_height="wrap_content"
        android:textSize="?attr/status_text_medium"
        app:layout_constraintBottom_toBottomOf="@id/status_reply"
        app:layout_constraintStart_toEndOf="@id/status_reply"
        app:layout_constraintTop_toTopOf="@id/status_reply" />
```